azure-pipelines.yml

```diff
@@ -443,7 +443,7 @@ stages:
       - task: Maven@3
         displayName: 'Maven verify, UTs with SonarCloud'
         inputs:
-          goals: 'verify sonar:sonar'
```

Comment:
this was added in #3018, must verify that code coverage isn't lost after this change